qdrant/oneof_factory.go

```diff
@@ -362,6 +362,33 @@ func NewVectorInputMulti(vectors [][]float32) *VectorInput {
 	}
 }
 
+// Creates a *VectorInput instance for a document.
+func NewVectorInputDocument(document *Document) *VectorInput {
+	return &VectorInput{
+		Variant: &VectorInput_Document{
+			Document: document,
+		},
+	}
+}
+
+// Creates a *VectorInput instance for an image.
+func NewVectorInputImage(image *Image) *VectorInput {
+	return &VectorInput{
+		Variant: &VectorInput_Image{
+			Image: image,
+		},
+	}
+}
+
+// Creates a *VectorInput instance for an inference object.
+func NewVectorInputObject(object *InferenceObject) *VectorInput {
+	return &VectorInput{
+		Variant: &VectorInput_Object{
+			Object: object,
+		},
+	}
+}
+
```

Comment:
We should hide these until we release the inference feature.

Could you comment these three out? That'd be good enough imo.